### DISCUSSION TITLE
finding a specific study in history

### DISCUSSION THREAD

**Discussion by TessadeBie** [asreview/asreview#1786] [Q&A]
Hello,

During the screening process another researcher conducted a quality check. after this check I want to change two studies that I identified as irrelevant to relevant. 
I want to search for these specific studies in the history, but I can't find a way to do this. Since I have to search through around 900 studies, I hope someone can help me. Does anyone have a tip in how to find specific studies in the history?

**Comment by J535D165** [comment]
Currently, there is no good way to do this. For now, the only solution I see is to export the results (and search for the study in a reference manager or spreadsheet reader/editor). 

I made a feature request, and we will implement it in the future.

**Comment by Rensvandeschoot** [comment]
Another workaround would be to use the search function of your browser (ctrl_f for Windows).

**Reply by TessadeBie** [reply]
thank you, that's how I did it.
